accepted/2021/utf8/requirements/README.md

```diff
@@ -0,0 +1,26 @@
+# Requirements
+
+**DRAFT**
+
+**PM** [Immo Landwerth](https://github.com/terrajobst)
+
+## [UTF-8 Strings should be pre-validated](validation.md)
+
+- allow us to do better optimizations, for example when enumerating runes
+- it's consistent with other ecosystems
+- better safety and reliability
+
+## String/UTF-8 String literals should be in read-only memory (read-only section of PE file)
+
+- allows to share strings between processes
+
+## UTF-8 Strings should be zero-terminated
```

Comment:
This is implementation detail, not a requirement. The requirement should be "Performant managed/native interop for UTF-8 strings".

Reply:
Since I'm changing the whole doc into `Areas of investigation` I'll leave the title as is (in form of question) but use this sentence instead of first bullet point since it captures this better